api/app/tests/apps/signals/factories.py

```diff
@@ -214,3 +215,12 @@ class DepartmentFactory(factory.DjangoModelFactory):
 
     class Meta:
         model = Department
+
+
+class NoteFactory(factory.DjangoModelFactory):
+    text = fuzzy.FuzzyText(length=100)
+    created_by = factory.Sequence(lambda n: 'veelmelder{}@example.com'.format(n))
+    _signal = factory.SubFactory('tests.apps.signals.factories.SignalFactory', status=None)
```

Comment:
This factory does not have to pass through a `status=None`.

Reply:
fixed